src/main/java/org/jenkinsci/plugins/configfiles/properties/security/HasPropertyCredentialMappings.java

```diff
@@ -0,0 +1,11 @@
+package org.jenkinsci.plugins.configfiles.properties.security;
+
+import java.util.List;
+
+public interface HasPropertyCredentialMappings {
+
+    public abstract List<PropertiesCredentialMapping> getPropertiesCredentialMappings();
```

Comment:
it's an interface. not sure we need `public abstract`

Reply:
Correct, I kept this way because this is how it has been implemented on maven.
It is now removed for properties